packages/ocamlfuse/ocamlfuse.2.7.1-cvs8/opam

```diff
@@ -0,0 +1,50 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "OCaml bindings for FUSE (Filesystem in UserSpacE)"
+description: """
+This is a binding to FUSE for the OCaml programming language, enabling
+you to write multithreaded filesystems in the OCaml language. It has
+been designed with simplicity as a goal, as you can see by looking at
+example/fusexmp.ml. Efficiency has also been a separate goal. The
+Bigarray library is used for read and writes, allowing the library to
+do zero-copy in OCaml land."""
+maintainer: ["Alessandro Strada <alessandro.strada@gmail.com>"]
+authors: [
+  "Vincenzo Ciancia"
+  "Olaf Hering <olaf@aepfle.de>"
+  "Alessandro Strada <alessandro.strada@gmail.com>"
+]
+license: "GPL-1.0-or-later"
+bug-reports: "https://github.com/astrada/ocamlfuse/issues"
+depends: [
+  "dune" {>= "3.7"}
+  "ocaml" {>= "4.02.3"}
```

Comment:
See failure above
```suggestion
  "ocaml" {>= "4.02.3" & < "5.0"}
```

Reply:
The problem seems to be caused by camlidl. It should be solved by this PR: https://github.com/xavierleroy/camlidl/pull/25. When it's merged, it should fix the error on macos with ocaml 5.0. The purpose of this new version is indeed supporting ocaml 5.0.